### ISSUE TITLE
Cluster Restore From Backup Fails

### ISSUE THREAD

**Issue by nasali** [flynn/flynn#2837]
Using the latest nightly build:
```
ubuntu@flynn0:~$ sudo CLUSTER_DOMAIN=mydomain.com flynn-host bootstrap --min-hosts 1 --discovery https://discovery.flynn.io/clusters/f33ad9d8-0e71-45a3-9841-995f77e74998 --from-backup ~/backup.tar.gz 
19:16:43.065726 check online-hosts
19:16:43.131174 run-app discoverd
19:16:43.403568 run-app flannel
19:16:43.805715 wait-hosts wait-hosts
19:16:46.309396 run-app postgres
19:16:49.755639 wait postgres-wait
19:16:52.344130 restore-postgres restore
19:45:20.413230 run-app controller
19:45:21.091634 wait-controller wait-controller
19:45:21.782059 run-app blobstore
19:45:22.785502 wait blobstore-wait
19:50:22.787517 wait blobstore-wait error: bootstrap: timed out waiting for http://blobstore.discoverd, last response 200
ubuntu@flynn0:~$ 
```

Here's the [debug gist](https://gist.github.com/anonymous/e71e957876d2e409a1d1c30d1bc5b806)

Thanks!

**Comment by titanous** [comment]
I see the bug, will fix.

**Comment by titanous** [comment]
Fixed in #2838.
